public/index.html

```diff
@@ -18,5 +18,20 @@
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
+    <script>
+      if ('serviceWorker' in navigator) {
```

Comment:
Interesting about how this is something that is applied straight into the index.html